packages/core/src/Search.cpp

```diff
@@ -343,11 +358,11 @@ void Search::Implementation::PathSet::add_path(const std::string& path)
   if(path.empty())
     return;
 
-  if(path.at(0) != '/')
+  if (IS_NOT_ABSOLUTE_PATH(path))
```

Comment:
Maybe I'm very "tikismikis" for this comment:
```suggestion
  if(IS_NOT_ABSOLUTE_PATH(path))
```

Reply:
Done